soak/README.md

```diff
@@ -189,17 +194,18 @@ Follow the commands below to install this helm chart.
 * Command
     ```bash
     go-to-soak \
+    && jq -n --arg CONTROLLER_CHART_RELEASE_NAME $CONTROLLER_CHART_RELEASE_NAME --arg SERVICE_NAME $SERVICE_NAME '{prometheus: {prometheusSpec: { additionalScrapeConfigs: [{job_name: "ack_controller", static_configs:[{ targets: ["\($CONTROLLER_CHART_RELEASE_NAME)-ack-\($SERVICE_NAME)-controller-metrics:8080"] }]}]}}}' | yq e -P > prometheus-values.yaml \
```

Comment:
Have you tried that this works? I think the DNS will change because controller is now installed in a different namespace than prometheus operator.
I think it will require ACK controller namespace in DNS query.

Reply:
That is a great point. I will verify that!